extensions/HuggingFace/python/src/aiconfig_extension_hugging_face/local_inference/image_2_text.py

```diff
@@ -248,18 +249,16 @@ def validate_and_retrieve_images_from_attachments(prompt: Prompt) -> list[Union[
     for i, attachment in enumerate(prompt.input.attachments):
         validate_attachment_type_is_image(prompt.name, attachment)
 
-        input_data = attachment.data
```

Comment:
Nit: We're calling `attachment.data` multiple times, so why not keep this? In L257 we can call it `image_source_value` or something

Reply:
shipping this as is; will look into this next time I touch this as that may be soon